src/pages/index.js

```diff
@@ -0,0 +1,15 @@
+import Head from 'next/head';
+
+export default function Home() {
+  return (
+    <>
+      <Head>
+        <title>Beeus</title>
+        <meta name="description" content="Beeus" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
```

Comment:
this `Head` children should be in the `_app` or `_document` because they're common to all pages, if we need to change the Title to another page just pass de component again with

```suggestion
      <Head>
        <title>Beeus New Title</title>
      </Head>
```

Reply:
Kept title configuration in child classes due to "No Title in Document Head" alert

Reply:
![image](https://user-images.githubusercontent.com/75763403/227733079-a4b0d50d-57e2-41bf-874a-b64a7b878766.png)

where?